scripts/build-installer-unix.sh

```diff
@@ -0,0 +1,150 @@
+while test $# -ge 1
+do case "$1" in
+           --fast-impure )                               fast_impure=true;;
+           --build-id )       arg2nz "build identifier";    build_id="$2"; shift;;
+           --travis-pr )      arg2nz "Travis pull request id";
+                                                           travis_pr="$2"; shift;;
+           --nix-path )       arg2nz "NIX_PATH value";
+                                                     export NIX_PATH="$2"; shift;;
+           --upload-s3 )                                   upload_s3=t;;
+           --test-install )                             test_install=t;;
+
+           ###
+           --verbose )        echo "$0: --verbose passed, enabling verbose operation"
+                                                             verbose=t;;
+           --quiet )          echo "$0: --quiet passed, disabling verbose operation"
+                                                             verbose=;;
+           --help )           usage;;
+           "--"* )            usage "unknown option: '$1'";;
+           * )                break;; esac
+   shift; done
+
+set -e
+if test -n "${verbose}"
+then set -x
+fi
+
+mkdir -p ~/.local/bin
+
+export PATH=$HOME/.local/bin:$PATH
+export DAEDALUS_VERSION=${daedalus_version}.${build_id}
+export SSL_CERT_FILE=$NIX_SSL_CERT_FILE
+
+test -d node_modules/daedalus-client-api/ -a -n "${fast_impure}" || {
+        retry 5 curl -o daedalus-bridge.tar.xz \
+              https://s3.eu-central-1.amazonaws.com/cardano-sl-travis/daedalus-bridge-${os}-${cardano_branch}.tar.xz
+        mkdir -p node_modules/daedalus-client-api/
+        du -sh  daedalus-bridge.tar.xz
+        tar xJf daedalus-bridge.tar.xz --strip-components=1 -C node_modules/daedalus-client-api/
+        rm      daedalus-bridge.tar.xz
+        cd node_modules/daedalus-client-api
+              mv log-config-prod.yaml cardano-node cardano-launcher ../../installers
+        cd ../..
+        strip installers/cardano-node installers/cardano-launcher
+        rm -f node_modules/daedalus-client-api/cardano-*
+}
+echo "cardano-sl build id is $(cat node_modules/daedalus-client-api/build-id)"
+
+test $(ls node_modules/ | wc -l) -gt 100 -a -n "${fast_impure}" ||
+        nix-shell --run "npm install"
+
+test -d "release/darwin-x64/Daedalus-darwin-x64" -a -n "${fast_impure}" || {
+        nix-shell --run "npm run package -- --icon installers/icons/256x256"
+        echo "Size of Electron app is $(du -sh release)"
+}
+
+test -n "$(which stack)"     -a -n "${fast_impure}" ||
+        retry 5 bash -c "curl -L https://www.stackage.org/stack/${os}-x86_64 | \
+                         tar xz --strip-components=1 -C ~/.local/bin"
+
+cd installers
+    if test "${travis_pr}" = "false" -a "${os}" != "linux" # No Linux keys yet.
+    then retry 5 nix-shell -p awscli --run "aws s3 cp --region eu-central-1 s3://iohk-private/${key} ${key}"
+    fi
+    stack --no-terminal --nix build --exec make-installer --jobs 2
```

Comment:
This used to have `appveyor-retry call` prepended, why is it left out? It's important because there are transient errors happening: DEVOPS-104